kymatio/scattering1d/utils.py

```diff
@@ -223,8 +188,10 @@ def compute_meta_scattering(J, Q, max_order=2):
             The tuples indexing the corresponding scattering coefficient
             in the non-vectorized output.
     """
-    sigma_low, xi1s, sigma1s, j1s, xi2s, sigma2s, j2s = \
-        calibrate_scattering_filters(J, Q)
+    sigma0 = 0.1
```

Comment:
Same.

Reply:
@janden should SIGMA0 be a global variable?

Reply:
addressed in 84c77d8